usr.sbin/bsnmpd/modules/snmp_wlan/snmp_wlan.3

```diff
@@ -64,7 +64,7 @@ Any optional parameters may be set via the
 and
 .Va wlanIfaceLocalAddress
 columns.
-To finally create the interface in the system, a SET with value of active(1) to
+To finally create the interface in the system, a SET with value of active = 1 to
```

Comment:
I don't think this is correct, it's snmp enum name and its value in parentheses.

Reply:
In that case it needs to be put into syntax that guards it from being turned into a man page link. such as a definition line, or a function call

Reply:
> In that case it needs to be put into syntax that guards it from being turned into a man page link. such as a definition line, or a function call

There is no macro before "active(1)", where/how exactly is it turned to a link?

Reply:
automatically.

Reply:
> automatically.

I have no idea what you mean by that, but wherever it is done should be fixed instead, this is a proper mdoc man page, so only explicit link macro should be turned into a link.

Reply:
I agree.
But sometimes we do it for… reasons
see https://github.com/freebsd/freebsd-src/blob/47a6fb9d5a2ebec12114a604053ffbd2929f0021/share/man/man9/domain.9#L122-L152

Reply:
> I agree. But sometimes we do it for… reasons see
> 
> https://github.com/freebsd/freebsd-src/blob/47a6fb9d5a2ebec12114a604053ffbd2929f0021/share/man/man9/domain.9#L122-L152

That's different, literal block output in your example is to keep formatting.  And in this case there's nothing looking like a macro so there's no need to escape or otherwise obfuscate the text.

Reply:
Just checked the `mandoc -Thtml` output, it correctly does NOT create a link here:
```
<dl class="Bl-tag">
  <dt id="wlanInterfaceTable"><var class="Va">wlanInterfaceTable</var></dt>
  <dd>The table is used for creation and deletion of virtual wireless
      interfaces. To add a new interface, a SET should be executed on the
      <var class="Va">wlanIfaceName</var> column with value the desired name of
      the interface. Next the parent interface must be set via
      <var class="Va">wlanParentIfName</var> column. Any optional parameters may
      be set via the <var class="Va">wlanIfaceOperatingMode</var>,
      <var class="Va">wlanIfaceFlags</var>, <var class="Va">wlanIfaceBssid</var>
      and <var class="Va">wlanIfaceLocalAddress</var> columns. To finally create
      the interface in the system, a SET with value of active(1) to
      <var class="Va">wlanIfaceStatus</var> column should be executed. To
      destroy a wireless interface a SET with value of destroy(6) to the
      relevant <var class="Va">wlanIfaceStatus</var> column should be
    executed.</dd>
```